content/en/docs/components/model-registry/overview.md

```diff
@@ -0,0 +1,35 @@
++++
+title = "Overview"
+description = "An overview for Kubeflow Model Registry"
+weight = 10
+                    
++++
+
+{{% alpha-status
+  feedbacklink="https://github.com/kubeflow/model-registry" %}}
+
+## What is Model Registry ?
+
+A model registry is an important component in the life cycle of AI/ML models, an integral component for any MLOps platform and for ML workflows.
+
+A model registry provides a central index for ML model developers to index and manage models, versions, and ML artifacts metadata.
+It fills a gap between model experimentation and production activities.
+It provides a central interface for all stakeholders in the ML lifecycle to collaborate on ML models.
+
+<img src="/docs/components/model-registry/images/index-mr-loop.png"
+  alt="Model Registry MLOps loop"
+  class="mt-3 mb-3">
+
+DevOps, Data Scientists, and developers need to collaborate with other users in the ML workflow to get models into production.
+Data scientists need an efficient way to share model versions, artifacts and metadata with other users that need to access to those models as part of the MLOps workflow.
+
+<!--
+## Architecture
+
+tbd, take from Proposal
+-->
+
+## Next steps
+
+Follow the [getting-started guide](/docs/components/model-registry/getting-started/)
```

Comment:
Can you fix the text?

Reply:
![image](https://github.com/kubeflow/website/assets/1699252/3281d506-4f24-4b6d-a456-56fc50b1d436)

not sure I understood your request @rimolive ... which part need fixing? 🤔

Reply:
Maybe `getting-started` should not have the `-` in the text, but it's just a nit thing. It's up to you.